### PR TITLE
Welcome Page

### LINKED ISSUE #96: Welcome page and Parity download links
From: https://github.com/paritytech/parity-extension/pull/95#issuecomment-323530428
screenshot from 2017-08-23 16-18-23
screenshot from 2017-08-23 16-18-17

### PR DESCRIPTION
**Pull request by tomusdrw**
Closes #96

### CODE REVIEW

File: welcome/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Parity Ethereum Integration | Parity Technologies</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <script src="./index.js" async></script>
```
Comment (ngotchac) [inline welcome/index.html]: Can't we have the CSS loaded here ? So there is no FOUC.

Comment (tomusdrw) [reply welcome/index.html]: The resources are loaded from the chrome extension, so it should work really fast and you never should see any UC.
I have inlined the CSS to address that though. (Having a standard css file would require some webpack configuration that I don't think is worth it)

**Comment by ngotchac** [comment]
I suggest something around these lines : 
```diff
diff --git a/webpack.config.js b/webpack.config.js
index ddf8816..78509a9 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -50,6 +50,11 @@ module.exports = {
   module: {
     rules: [
       {
+        test: /welcome\.css/,
+        loader: 'file-loader?name=[path][name].[ext]'
+      },
+
+      {
         test: /\.js$/,
         exclude: /(node_modules)/,
         use: 'babel-loader'
@@ -59,13 +64,13 @@ module.exports = {
         test: /\.html$/,
         use: [
           'file-loader?name=[path][name].[ext]',
-          'extract-loader?publicPath=./',
-          'html-loader'
+          'extract-loader?publicPath=../',
+          'html-loader?attrs[]=img:src&attrs[]=link:href'
         ]
       },
       {
         test: /\.css$/,
-        exclude: [ /node_modules/, /welcome.welcome\.css/ ],
+        exclude: [ /node_modules/, /welcome\.css/ ],
         use: [
           'style-loader',
           'css-loader?modules&sourceMap&importLoaders=1&localIdentName=[name]_[local]_[hash:base64:5]',
@@ -75,7 +80,7 @@ module.exports = {
 
       {
         test: /\.css$/,
-        include: [ /node_modules/, /welcome.welcome\.css/ ],
+        include: [ /node_modules/ ],
         use: [
           'style-loader',
           'css-loader'
diff --git a/welcome/index.html b/welcome/index.html
index 120f82b..70296cc 100644
--- a/welcome/index.html
+++ b/welcome/index.html
@@ -4,6 +4,7 @@
     <title>Parity Ethereum Integration | Parity Technologies</title>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link rel="stylesheet" href="./welcome.css">
     <script src="./index.js" async></script>
 </head>
 <body>
diff --git a/welcome/index.js b/welcome/index.js
index 3e4a216..7489b14 100644
--- a/welcome/index.js
+++ b/welcome/index.js
@@ -15,8 +15,6 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 import './index.html';
-
-import './welcome.css';
 import '../assets/fonts/Roboto/font.css';
 
 import { browser } from '../shared';
```

**Comment by tomusdrw** [comment]
Updated :+1:

File: welcome/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import './index.html';
+
+import './welcome.css';
```
Comment (ngotchac) [inline welcome/index.js]: This shouldn't be needed anymore :)

**Review by ngotchac — CHANGES_REQUESTED** [review]

**Comment by tomusdrw** [comment]
Updated :+1:  :)

**Review by ngotchac — APPROVED** [review]

**Pull request merged**